src/components/select/select.js

```diff
@@ -135,6 +137,18 @@ angular.module('material.components.select', [
  *     </md-select>
  *   </md-input-container>
  * </hljs>
+ * Multiselect with a custom separator
+ *
+ * When a developer needs to add a seperator with multi select, adding to the attribute `selection-separator-char` or
+ `selection-separator-spacing` with options. The example below demonstrate a new line for after each selected item.
+ *
+ * <hljs lang="html">
+ *   <md-input-container>
+ *     <md-select ng-model="someModel" selection-separator-char="none" selection-separator-spacing="break">
```

Comment:
For the line break example, let's just omit the `selection-separator-char="none"` and verify that the code works in that case. We'll probably want a unit test for that.